src/Microsoft.AspNetCore.Razor.Design/build/netstandard2.0/Microsoft.AspNetCore.Razor.Design.props

```diff
@@ -47,6 +42,4 @@
     <ProjectCapability Include="DotNetCoreRazorConfiguration"/>
   </ItemGroup>
 
-  <UsingTask TaskName="Microsoft.AspNetCore.Razor.Tasks.RazorGenerate" AssemblyFile="$(_RazorTaskAssembly)" />
```

Comment:
This goes in the targets since we control when the target gets loaded and don't have to worry about ordering.